src/app/content/components/Page.tsx

```diff
@@ -15,6 +15,12 @@ export default class PageContent extends Component<PropTypes> {
       .replace(/<cnx-pi.*>[\s\S]*<\/cnx-pi>/g, '');
   }
 
+  public componentDidUpdate(prevProps: PropTypes) {
+    if (window && prevProps.content !== this.props.content) {
+      window.scrollTo(0, 0);
```

Comment:
It seems like this causes a link containing a hash to break. Example: http://localhost:3000/books/Ax2o07Ul/pages/m0l_Rsl5#import-auto-id2932439

Reply:
how are you seeing it break? the figure links work for me